### DISCUSSION TITLE
Traffic-Monitor for API-Manager Administrator + operator of API Gateway Manager is invalid

### DISCUSSION THREAD

**Discussion by lyr3051** [Axway-API-Management-Plus/apigateway-openlogging-elk#194] [Q&A]
Hello~
When I use Administrator of API Gateway Manager to see the traffic , all requests I called are successfully extract from Elasticsearch.
<img width="1440" alt="截圖 2022-03-22 下午3 26 30" src="https://user-images.githubusercontent.com/43023893/159430725-c9ff110b-5c2d-46cc-a836-3f3a511269be.png">

However, when I try to use the user "fred" with API Manager Administrator and the operator of API Gateway Manager to see the traffic of API, I can't see any traffic in the Traffic Monitor.
<img width="1439" alt="截圖 2022-03-22 下午3 35 10" src="https://user-images.githubusercontent.com/43023893/159430415-07690300-5bf6-46bc-8b44-05cf3e166717.png">
<img width="787" alt="截圖 2022-03-22 下午3 08 54" src="https://user-images.githubusercontent.com/43023893/159429971-b1f5dacb-81d6-46f6-836c-48cd49533387.png">
<img width="727" alt="截圖 2022-03-22 下午3 09 22" src="https://user-images.githubusercontent.com/43023893/159429982-d050df99-67a8-451f-ae5c-d99b39d172ce.png">

I check the log of apibuilder4elastic container, the request to extract the traffic is return 500 error.
<img width="1440" alt="截圖 2022-03-22 下午3 10 41" src="https://user-images.githubusercontent.com/43023893/159431199-794b2f10-170f-485b-81f9-be2ad82859bf.png">

Is this issue becasue of my misconfiguration?

**Comment by cwiechmann** [comment]
Hi @lyr3051, 

You see the error message in the response for the API-Builder REST-API-Call using the Browser-Development Tools as illustrated:
![See API-Builder response](https://user-images.githubusercontent.com/16292002/159440300-2502c3aa-f1f2-4657-b1b1-ac03f003e748.gif)

Please share the error message.

**Reply by lyr3051** [reply]
Hi @cwiechmann 
The error message please find the below screenshot:
<img width="1326" alt="image" src="https://user-images.githubusercontent.com/43023893/160086543-ce51f598-9d7f-486d-b56c-e24769218fae.png">

When I promoted the user "fred" to the API Admin role, he was automatically added to the community organization.
<img width="1057" alt="image" src="https://user-images.githubusercontent.com/43023893/160087151-4105db4d-d3ce-449d-bdbb-bad7e8a43f0d.png">

**Reply by cwiechmann** [reply]
If fred becomes an API-Admin, he has no "Organization" restriction anymore and should be able to see the entire traffic for APIs managed by the API-Manager. 
Keep in mind, that information about the user is cached in APIBuilder4Elastic as well, to avoid looking up users every single time.

**Reply by lyr3051** [reply]
Hi @cwiechmann 
The issue is solved when I enable the API Development in Community Organization, but the API Development is disable by default.
<img width="1439" alt="image" src="https://user-images.githubusercontent.com/43023893/191488676-aded97ba-5337-44e2-809f-8d48dfcdedc6.png">
Is this the right configuration to see the traffic in Traffic Monitor when the user of role is API Manager Administrator and API Gateway Operator?
